apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/InstitusjonsoppholdOMS.tsx

```diff
@@ -0,0 +1,185 @@
+import { IBehandlingReducer } from '~store/reducers/BehandlingReducer'
```

Comment:
Er disse tingene så forskjellig på BP og OMS at vi trenger helt egne komponenter for hver av dem?

Reply:
Du tenker heller at man benytter BP og OMS tekster i samme komponent?